datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SdxCustomClusterRequest.java

```diff
@@ -0,0 +1,59 @@
+package com.sequenceiq.sdx.api.model;
+
+import java.io.IOException;
+
+import org.apache.commons.lang3.tuple.Pair;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.request.image.ImageSettingsV4Request;
+
+public class SdxCustomClusterRequest extends SdxClusterRequestBase {
+    private ImageSettingsV4Request imageSettingsV4Request;
+
+    public ImageSettingsV4Request getImageSettingsV4Request() {
+        return imageSettingsV4Request;
+    }
+
+    public void setImageSettingsV4Request(ImageSettingsV4Request imageSettingsV4Request) {
+        this.imageSettingsV4Request = imageSettingsV4Request;
+    }
+
+    public String getRuntime() {
+        return null;
+    }
+
+    @Override
+    public String toString() {
+        String baseString = super.toString();
+        String additionalString = ", imageSettingsV4Request=" + imageSettingsV4Request +  '}';
+        String ownString = baseString.replace("SdxClusterRequestBase", "SdxCustomClusterRequest");
+        return ownString.replace("}", additionalString);
```

Comment:
Like in `SdxClusterRequest`, I do not think it is a good idea to resort to string-wrestling.

Reply:
I will update accordingly.